user_manual/ios_app.rst

```diff
@@ -71,11 +73,12 @@ files, click the Share button to see who you have shared with, and to remove
 shares.
 
 You may also create Federation shares from your iOS app. If you are sharing 
-files with ownCloud 9.x users, you can enter their username and server URL in 
-the User and Groups dialog to create a direct share; for example 
-freda@example.com/owncloud. Your recipient will receive a notification asking 
-if they wish to accept the share, and then all they have to do is click an 
-"Accept" button.
+files with ownCloud 9.x users, you can enter their username (auto-completion is 
+supported for Federation sharing) and server URL in the User and Groups dialog 
+to create a direct share; for example freda@example.com/owncloud. (The exact 
+Federation share link is on every ownCloud user's Personal page in the ownCloud 
+Web interface.) Your recipient will receive a notification asking if they wish 
```

Comment:
It would also be cool to see the user's Federation share URL in the app, just like on the Personal page in the Web GUI.